platforms-maturity-model/v1/index.md

```diff
@@ -0,0 +1,482 @@
+
+### Interface
+<h4 style="color:gray"><i>How do users interact with and consume platform capabilities?</i></h4>
+
+The interfaces provided by platforms affect how users interact with these platform offerings to provision, manage, and observe capabilities. Interfaces can include ticketing systems, project templates, and graphical portals as well as automatable APIs and command-line (CLI) tools.
+
+Key characteristics of an interface include how discoverable and user-friendly it is during key user journeys like initial request, maintenance, or incident triage. Higher levels of maturity here reflect more integrated, consistent, automated, and supported interfaces.
+
+##### Level 1, Provisional - Custom processes
+
+A collection of varying processes exists to provision different capabilities and services, but the consistency of the interface is not considered. Custom tailor-made processes address the immediate needs of individuals or teams and are reliant on manual intervention even if the provider uses some automated implementation scripts.
+
+Knowledge of how to request these solutions is shared from person to person. The process for requesting a service lacks standardization and consistency. Provisioning and using a platform service likely requires deep support from the capability provider.
+
+Lack of central requirements and standards makes this level appropriate when the company has not yet identified and documented expectations. It provides early stage companies and efforts the freedom for teams to evolve processes and capabilities to their needs, allowing them to deliver more quickly.
+
+###### Characteristics:
+
+* User interaction is not a key topic of discussion and rarely (if ever) are interactions tested during design and delivery of new capabilities
+* Capabilities are mainly provided through manual requests, though providers may choose to automate some or all of the activities necessary to provision a user request
+* Requests that are on the face “simple” become complex due to finding out the right process to follow
+* Sometimes a process appears to be sanctioned, but users run into issues when a different department or team gets involved
+
+###### Examples Scenarios:
+
+* An application team wants to performance test their new change. To do this, they want an isolated environment that contains enough test data to get an accurate performance read. The last time they had this request a former teammate was able to get access to an environment, but they have since moved on and no one knows how to recreate it. In the end, they are connected to an engineer on the infrastructure team who is able to provision them an environment in a few days.
+* A team in the exploratory phases of product development uses a bespoke process to provision a new cloud service prior to validating the solution warrants further investment.
```

Comment:
```suggestion
* A team in the exploratory phases of product development uses a bespoke process to provision a new cloud service prior to validating the solution, which warrants further investment.
```
not sure, maybe something else is meant originally here or some other word got lost